snuba/datasets/querylog_processor.py

```diff
@@ -0,0 +1,72 @@
+from typing import Any, Mapping, Optional, Sequence, Union
+import uuid
+
+from snuba.processor import MessageProcessor, ProcessedMessage, ProcessorAction
+
+
+class QuerylogProcessor(MessageProcessor):
+    def get_request(self, request: Mapping[str, Any]) -> str:
+        return str({k: v for k, v in sorted(request.items())})
```

Comment:
I think it would be worth to format this int oJSON instead of just serializing the mapping, Clickhosue has some json processing functions https://clickhouse.tech/docs/en/query_language/functions/json_functions/ that would make querying this dataset easier